librz/analysis/var.c

```diff
@@ -1171,6 +1171,11 @@ static void extract_stack_var(RzAnalysis *analysis, RzAnalysisFunction *fcn, RzA
 		if (*sign == '-') {
 			addend = -addend;
 		}
+		if (addend == 0 && op->direction != RZ_ANALYSIS_OP_DIR_READ && op->direction != RZ_ANALYSIS_OP_DIR_WRITE) {
```

Comment:
Extract this long condition into a small inline function: https://github.com/rizinorg/rizin/blob/dev/DEVELOPERS.md#c

Reply:
Maybe for other few conditions in the same function too.